addons/website_slides/models/slide_channel.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from copy import copy
```

Comment:
where is this import used ?

Reply:
Visual Studio Code sneakily added it there when I typed `copy=False`... will be more careful next time